internal/pkg/initialize/workload.go

```diff
@@ -60,14 +60,15 @@ type Prog interface {
 
 // WorkloadProps contains the information needed to represent a Workload (job or service).
 type WorkloadProps struct {
-	App            string
-	Type           string
-	Name           string
-	DockerfilePath string
-	Image          string
-	Platform       manifest.PlatformArgsOrString
-	Topics         []manifest.TopicSubscription
-	Queue          manifest.SQSQueue
+	App                     string
+	Type                    string
+	Name                    string
+	DockerfilePath          string
+	Image                   string
+	Platform                manifest.PlatformArgsOrString
+	Topics                  []manifest.TopicSubscription
+	Queue                   manifest.SQSQueue
+	PrivateOnlyEnvironments []string
```

Comment:
Maybe in the future if we are doing more "magic" setting to the manifest (per environment), we could further extend this field to something like

```golang
Environments map[string]EnvOverrideSetting
```

and then unwrap it in the manifest package. But I think right now it is good enough.